packages/host/tests/acceptance/code-submode/field-playground-test.gts

```diff
@@ -1018,13 +1017,6 @@ module('Acceptance | code-submode | field playground', function (_hooks) {
       });
       setupUserSubscription(matrixRoomId);
 
-      let realmServerService = this.owner.lookup(
-        'service:realm-server',
-      ) as RealmServerService;
-      personalRealmURL = `${realmServerService.url}testuser/personal/`;
-      additionalRealmURL = `${realmServerService.url}testuser/aaa/`; // writeable realm that is lexically before the personal realm
-      setActiveRealms([additionalRealmURL, personalRealmURL]);
```

Comment:
it's too late to set this here. the state that this is set in has already been read and realm matrix rooms have already been created.